simhalt.h

```diff
@@ -786,6 +786,10 @@ class haltestelle_t
 	 */
 	void get_short_freight_info(cbuffer_t & buf) const;
 
+	void get_transfers_info(cbuffer_t& buf) const;
+	
+	void get_percent_info(cbuffer_t& buf) const;
```

Comment:
Please make the name more concrete. (Please suggest a better name than `get_waiting_occupancy_info` if you have.)
```suggestion
	void get_waiting_occupancy_info(cbuffer_t& buf) const;
```